packages/cozy-mespapiers-lib/src/components/Papers/PapersList.jsx

```diff
@@ -52,6 +52,10 @@ const PapersList = ({ papers, isLast }) => {
       ),
     [actionVariant, client, addMultiSelectionFile, popModal, pushModal]
   )
+  const accountLogin = papers?.list?.[0]?.cozyMetadata?.sourceAccountIdentifier
+  const account = accounts?.find(
+    account => account?.auth?.login === accountLogin
+  )
```

Comment:
why are we doing the find if there is no `sourceAccountIdentifier`?

Reply:
pas de raison particulière, je vais le conditionner